src/main/java/com/manardenza/entity/Room.java

```diff
@@ -0,0 +1,20 @@
+package com.manardenza.entity;
+
+import lombok.Data;
+
+@Data
+public class Room extends AbstractObject {
+
+    private String name;
+    private int person;
+    private int price;
+    private Reservation reservation;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "reservation" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1068)